### ISSUE TITLE
LDAP integration

### ISSUE THREAD

**Issue by anmol26s** [YunoHost-Apps/BicBucStriim_ynh#1]
After installation when I enter the username and password it gives error message saying:
Error loading page

I can not see any database for bbs in phpmyadmin. On the check page Apache web server, .htaccess, url rewriting show yellow symbol. GD version 2 has red symbol.
I have already installed  sqlite3, php5-gd, php5-intl.

**Comment by a1ex4** [comment]
Yes, after installing you need to login with user "admin" and password "admin", then create a new user. This is because I couldn't get the LDAP integration working, I don't know how to fetch the user password during the installation of the package.

**Comment by anmol26s** [comment]
Thanks 
Login with admin and admin worked.

**Comment by a1ex4** [comment]
LDAP clearly needs to be implemented, will see if I'm able to do it.

https://github.com/rvolz/BicBucStriim/tree/master/vendor/aura/auth
